src/util/config.cpp

```diff
@@ -912,6 +912,9 @@ bool configt::set(const cmdlinet &cmdline)
     ansi_c.preprocessor=ansi_ct::preprocessort::GCC;
   }
 
+  if(ansi_c.preprocessor == ansi_ct::preprocessort::GCC)
```

Comment:
Clang?

Reply:
Agreed, I'll try to cook one up tomorrow morning...